generators/ambassador/ambassador.go

```diff
@@ -222,13 +222,19 @@ func (g *Generator) Generate(opts *options.Options, spec *openapi3.T) (string, e
 						}
 					}
 
+					if rateLimitOpts.Group == "" {
```

Comment:
is it possible in the struct tags to set a default for this option rather than doing it here? It will make the code look a bit cleaner if we could?